cli/vm/vm.go

```diff
@@ -24,13 +25,20 @@ func NewCommands() []cli.Command {
 }
 
 func startVMPrompt(ctx *cli.Context) error {
+	var (
+		cfg config.Config
+		err error
+	)
+
 	if err := cmdargs.EnsureNone(ctx); err != nil {
 		return err
 	}
-
-	cfg, err := options.GetConfigFromContext(ctx)
-	if err != nil {
-		return cli.NewExitError(err, 1)
+	cfg = options.DefaultConfig
+	if ctx.String("config-file") != "" || ctx.String("config-path") != "" {
+		cfg, err = options.GetConfigFromContext(ctx)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
 	}
 	if ctx.NumFlags() == 0 {
```

Comment:
This path must be removed, we have default config for that. It should use in-memory DB.